content/finite-fields/en/finite-fields.md

```diff
@@ -8,9 +8,9 @@ Both of these issues can be handled seamlessly with a variant of arithmetic, whi
 
 ## Finite fields
 
-Given a prime number `p`, we can make a finite field with `p` elements by taking the set of integers $\set{0, 1, 2, …, p-1}$ and define addition and multiplication to be done modulo $p$. We’ll start by limiting ourselves to fields where the number of elements is a prime.
+Given a prime number `p`, we can make a finite field with `p` elements by taking the set of integers $\set{0, 1, 2, …, p-1}$ and define addition and multiplication to be done modulo $p$. We’ll start by limiting ourselves to fields where the number of elements is a [prime](https://en.wikipedia.org/wiki/Prime_number).
```

Comment:
For something as basic as a prime number, I'd rather not link it.

Reply:
It.. wouldn't take much space. But ok.